configuration/pih/pih-config-liberia-harper-jjdossen.json

```diff
@@ -1,38 +1,4 @@
 {
-
```

Comment:
I moved all these out of "pih-config-liberia-harper-jjdossen" and into "pih-config-harper"... both JJ Dossen and Kouka use the "pih-config-harper" so they will now have the exact same components enabled.

I also removed "MCH" from the list so that it won't be enabled.

Reply:
Maybe @BranfordTGbieor  wants MCH on Kouka for development?

Reply:
Thanks @lnball, but I'm not sure we would need that for now. I suggested the dev of the forms reach a particular stage before we enable them on kouka for testings purposes. Maybe in the next two weeks or so...